Sources/StevensonCore/Release.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import Vapor
+
+extension GitHubService {
+    public struct Release {
+        public let repository: Repository
+        public let branch: String
+        public let appName: String?
+        public let version: String
+
+        /// - Parameters:
+        ///   - repository: the GHRepo this release is for, ios or android
```

Comment:
I agree with `s/GHRepo/GitHub repo/`, but it can be any repo. I've separated the generic stuff (in `StevensonCore`) that should be agnostic from the repo you use it with, from the specific repos we use (that's why `RepoID` is in App and not StevensonCore, and one could add more repos on the future in the app config without having to change StevensonCore), so I wouldn't mention "ios or android" here.